tests/telemetry/test_telemetry.py

```diff
@@ -29,16 +28,16 @@ def test_config_db_parameters(duthost):
     else:
         d = get_dict_stdout(gnmi, certs)
         for key, value in d.items():
-            if str(key) == "client_auth" and str(value) == "true":
-                assert True, "Client_auth set to true"
-            if str(key) == "port" and str(value) == "50051":
-                assert True, "port is set to 50051"
-            if str(key) == "ca_crt" and str(value) == "/etc/sonic/telemetry/dsmsroot.cer":
-                assert True, "ca_crt is set to {}".format(str(value))
-            if str(key) == "server_key" and str(value) == "/etc/sonic/telemetry/streamingtelemetryserver.key":
-                assert True, "server_key is set to {}".format(str(value))
-            if str(key) == "server_crt" and str(value) == "/etc/sonic/telemetry/streamingtelemetryserver.cer":
-                assert True, "server_crt is set to {}".format(str(value))
+            if str(key) == "client_auth":
+                assert str(value) == "true", 'Client_auth should be set to true!'
+            if str(key) == "port":
+                assert str(value) == "50051", "Port should be set to 50051"
+            if str(key) == "ca_crt":
+                assert str(value) == "/etc/sonic/telemetry/dsmsroot.cer", "ca_crt  value should be set to /etc/sonic/telemetry/dsmsroot.cer"
+            if str(key) == "server_key":
+                assert str(value) == "/etc/sonic/telemetry/streamingtelemetryserver.key", "server_key  value should be set to  /etc/sonic/telemetry/streamingtelemetryserver.key"
+            if str(key) == "server_crt":
+                assert str(value) == "/etc/sonic/telemetry/streamingtelemetryserver.cer", "server_crt is set to /etc/sonic/telemetry/streamingtelemetryserver.cer"
```

Comment:
Suggest setting variables and reusing them in both the assert condition and message. This makes for easier and more error-free maintenance in the future. E.g.,

```
            if str(key) == "client_auth":
                client_auth_expected = "true"
                assert str(value) == client_auth_expected , "'client_auth' value is not '{}'".format(client_auth_expected)
            if str(key) == "port":
                expected_port = "50051"
                assert str(value) == expected_port, "'port' value is not '{}'".format(expected_port)
            ...
```

Reply:
Updated accordingly